### PR TITLE
Import X509 in PackageSigning 

### PR DESCRIPTION
**Pull request by neonichu**
This includes https://github.com/apple/swift-package-manager/pull/6216 right now so that we can run CI with both.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -730,9 +722,10 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // dependency version changes here with those projects.
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.2.2")),
         .package(url: "https://github.com/apple/swift-driver.git", .branch(relatedDependenciesBranch)),
-        .package(url: "https://github.com/apple/swift-crypto.git", .upToNextMinor(from: minimumCryptoVersion)),
+        .package(url: "https://github.com/apple/swift-crypto.git", .upToNextMinor(from: "2.3.0")),
         .package(url: "https://github.com/apple/swift-system.git", .upToNextMinor(from: "1.1.1")),
         .package(url: "https://github.com/apple/swift-collections.git", .upToNextMinor(from: "1.0.1")),
+        .package(url: "https://github.com/apple/swift-certificates.git", branch: "main"),
     ]
```
Comment (yim-lee) [inline Package.swift:729]: We don't need to add swift-asn1 here?

Comment (neonichu) [reply Package.swift:729]: Nope, SwiftPM doesn't do anything different based on direct vs. transitive dependency.

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by neonichu** [comment]
Windows build will fail here since it needs https://github.com/apple/swift/pull/64087

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by tomerd** [comment]
seems to work 🎉

**Review by tomerd — APPROVED** [review]

**Pull request merged**